ractor/src/port/output/tests.rs

```diff
@@ -158,6 +158,84 @@ async fn test_50_receivers() {
         .unwrap();
 }
 
+#[crate::concurrency::test]
+#[tracing_test::traced_test]
+async fn test_delivery() {
```

Comment:
This test consistently fails with the version on the main branch, and passes when `Drop` is removed